src/chrome/content/rules/Sinaimg.cn.xml

```diff
@@ -3,16 +3,18 @@
 
 	Invalid certificate:
 		u1.sinaimg.cn.cdngc.net
+		www.sinaimg.cn	( Uses Akamai outside the Chinese GFW, something else inside the GFW that doesn't get a mismatch. )
```

Comment:
I tested https://www.sinaimg.cn/cj/pc/2007-10-19/32/U453P31T32D36393F1538DT20071019102225.pdf through an US-CA tunnel and it seems to be working. I suppose [Sinaimg.cn.xml](https://github.com/EFForg/https-everywhere/blob/3b1aad6bf1791f03b00ce5c05ec524052a5f084d/src/chrome/content/rules/Sinaimg.cn.xml) can be updated accordingly.